src/Jaeger/Config.php

```diff
@@ -312,6 +344,10 @@ private function setConfigFromEnv()
             $this->config['tags'] = $_ENV['JAEGER_TAGS'];
         }
 
+        if (isset($_ENV['JAEGER_DISPATCH_MODE']) && !isset($this->config['dispatch_mode'])) {
+            $this->config['dispatch_mode'] = $_ENV['JAEGER_DISPATCH_MODE'];
```

Comment:
getenv ?

Reply:
The whole method used $_ENV. Do I need to update all of them?